src/services/crudServices.ts

```diff
@@ -0,0 +1,9 @@
+export let deleteUser = (users, id: number) => {
+    const index = users.findIndex(user => user.id == id);
+
+    if (index != -1) {
+        return users.splice(index, 1);
+    }
+
+    return {};

```

Comment:
se togli questo return lui tornerà undefined (void) in caso non entri nell if e poi nel router puoi mettere che ctx.body lo inizializza solo se if user

Reply:
io avevo provato a ritornare direttamente null o undefined e non funzionava

Reply:
ora provo

Reply:
non c'è bisogno, ma return undefined è 
`return`
da solo